bootstrap

```diff
@@ -16,6 +17,29 @@ main() {
 }
 
 
+# Apply user id and group id
+apply_permissions() {
+    if [[ -n "$USER_ID" && -n "$GROUP_ID" ]]; then
```

Comment:
Instead of checking if these are defined let's put defaults into the `defaults` file.
Something like
```
PUID=${PUID:-0}
PGID=${PGID:-0}
```
and use the same defaults in the `Dockerfile`.

Reply:
Sounds good to me. Please have a look at the updated branch.